client/src/pages/Profile.js

```diff
@@ -1,24 +1,153 @@
-import React from "react";
-import Typography from "@material-ui/core/Typography";
+import React, { useState } from "react";
 import Grid from "@material-ui/core/Grid";
-//Material-ui imports
-
+import Container from "@material-ui/core/Container";
+import Avatar from "@material-ui/core/Avatar";
+import Box from "@material-ui/core/Box";
+import AvatarImage from "./Profile/img/avatar.png";
+import Paper from "@material-ui/core/Paper";
+import { makeStyles } from "@material-ui/core/styles";
+import PageHeader from "../elements/PageHeader";
+import ProfileStats from "./Profile/ProfileStats";
+import ProfileProjects from "./Profile/ProfileProjects";
 import Background from "../elements/Background";
 import Navbar from "../components/Navbar";
+import ProfileSkills from "./Profile/ProfileSkills";
+import ProfileName from "./Profile/ProfileName";
+import EditIcon from "@material-ui/icons/Edit";
+import Button from "@material-ui/core/Button";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    position: "relative",
+  },
+  paper: {
+    textAlign: "center",
+    padding: "5vh",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+    marginTop: theme.spacing(12),
+    marginBottom: theme.spacing(12),
+  },
+  avatar: {
+    width: theme.spacing(18),
+    height: theme.spacing(18),
+    position: "absolute",
+    left: "50%",
+    padding: 0,
+    margin: 0,
+    transform: "translate(-50%, -50%)",
+    border: "3px solid #fff",
+  },
+  iconEdit: {
+    color: "gray",
+  },
+  button: {
+    margin: theme.spacing(1),
+  },
+}));
 
 const Profile = ({ state, dispatch }) => {
+  const classes = useStyles();
+  const [edit, setEdit] = useState(false);
+  const [name, setName] = useState("Steve Jobs");
+  const [about, setAbout] = useState("Apple Chief Officer");
+  const [files, setFiles] = useState([]);
+  const skills = [
+    { language: "JavaScript", level: "Advanced" },
+    { language: "Java", level: "Expert" },
+    { language: "PHP", level: "Expert" },
+    { language: "C++", level: "Beginner" },
+  ];
+  const projects = [
+    { title: "JS Code Review", link: "http://localhost:3000/" },
+    { title: "New Java Code", link: "http://localhost:3000/" },
+  ];
```

Comment:
skills and projects should be fetched from user. can make a new field in user schema for projects and it should have an array of of projects that has title, link and maybe an image for that project.

Reply:
I think for now, since there are no projects in the back-end, let's remove this hard coding and not show projects. Instead, let's just show skills, and fetch them from the user as Affeeq mentioned